src/reflection/FStar.Reflection.Basic.fs

```diff
@@ -65,20 +65,30 @@ let get_env () : Env.env =
   | Some e -> e
 
 (* private *)
-let inspect_aqual (aq : aqual) : aqualv =
-    match aq with
+let inspect_bqual (bq : bqual) : aqualv =
+    match bq with
     | Some (Implicit _) -> Data.Q_Implicit
     | Some (Meta t) -> Data.Q_Meta t
     | Some Equality
     | None -> Data.Q_Explicit
 
+let inspect_aqual (aq : aqual) : aqualv =
+    match aq with
+    | Some ({ aqual_implicit = true }) -> Data.Q_Implicit
+    | _ -> Data.Q_Explicit
+
```

Comment:
Note, the reflection API did not change. Both aqual and bqual are projected to aqualv.